packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -74,7 +74,7 @@ const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = (
             "&:not($carousel_sequenceIndicator__active)": {
                 "&:hover": {
                     "&::before": {
-                        opacity: "0.5",
+                        opacity: "0.9",
```

Comment:
should these opacities be based on anything?

Reply:
Possibly. I don't recognize a pattern that's in place yet though. This change was because the change was not perceivable, but the initial values came from experimentation with this particular usage over an image. I'm not sure what I would call this from a design systems perspective yet, but it's worth watching and considering making a setting.